core/src/autogluon/core/models/greedy_ensemble/ensemble_selection.py

```diff
@@ -45,14 +45,15 @@ def fit(self, predictions, labels, time_limit=None, identifiers=None):
         # if not isinstance(self.metric, Scorer):
         #     raise ValueError('Metric must be of type scorer')
 
-        self._fit(predictions=predictions, labels=labels, time_limit=time_limit)
+        self._fit(predictions=predictions, labels=labels, time_limit=time_limit, **kwargs)
         self._calculate_weights()
         logger.log(15, 'Ensemble weights: ')
         logger.log(15, self.weights_)
         return self
 
     # TODO: Consider having a removal stage, remove each model and see if score is affected, if improves or not effected, remove it.
-    def _fit(self, predictions, labels, time_limit=None):
+    def _fit(self, predictions, labels, time_limit=None, **kwargs):
```

Comment:
general coding practice:

I think it makes sense to not use kwargs here, since sample_weight is the only valid kwarg, and it isn't supposed to receive extra arguments beyond that.